variables.tf

```diff
@@ -0,0 +1,59 @@
+variable "environment" {
+  description = "Environment name. Will be used along with `project_name` as a prefix for all resources."
+  type        = string
+}
+
+variable "project_name" {
+  description = "Project name. Will be used along with `environment` as a prefix for all resources."
+  type        = string
+}
+
+variable "azure_location" {
+  description = "Azure location in which to launch resources."
+  type        = string
+}
+
+variable "registry_sku" {
+  description = "The registry SKU. To create Private Endpoints, 'Premium' is required."
+  type        = string
+  default     = "Basic"
```

Comment:
`Standard` is the default SKU

Reply:
In Terraform it doesn't actually have a default:

> (Required) The SKU name of the container registry. Possible values are Basic, Standard and Premium.

Basic is the lowest so went with that

Reply:
Doh! You're absolutely correct. (https://learn.microsoft.com/en-us/azure/container-registry/container-registry-skus)

This will not block us then :)